{
  "comments": [
    {
      "unresolved": true,
      "key": {
        "uuid": "2e01ddb9_a52db474",
        "filename": "/COMMIT_MSG",
        "patchSetId": 2
      },
      "lineNbr": 9,
      "author": {
        "id": 1161379
      },
      "writtenOn": "2021-03-22T08:03:02Z",
      "side": 1,
      "message": "I think that is intentional behavior?\nhttps://source.chromium.org/chromium/infra/infra/+/master:luci/appengine/swarming/swarming_rpcs.py;l\u003d550-552;drc\u003d76946843c79ab49a61b501cf8b3c89d075fbe9e5",
      "revId": "b2b70ab449353f15816867f1932800d11d66a66e",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "6ea72d31_cccdbd1e",
        "filename": "/COMMIT_MSG",
        "patchSetId": 2
      },
      "lineNbr": 9,
      "author": {
        "id": 1366072
      },
      "writtenOn": "2021-03-22T08:42:47Z",
      "side": 1,
      "message": "Note that _cancel_task_tx() is a part of task cancel request, which is triggered by users.\nI think users don\u0027t need to extend the deadline since bots should update it by pinging.\n\nThe tricky part is that _cancel_task_tx() is called in _bot_update_tx().\nhttps://source.chromium.org/chromium/infra/infra/+/master:luci/appengine/swarming/server/task_scheduler.py;l\u003d797-799;drc\u003dbb840bc41d72df96b0f84898edd5ab7ce46e9326\n\nBut `dead_after_ts` is already updated in _bot_update_tx(). So it looks redundant.\nhttps://source.chromium.org/chromium/infra/infra/+/master:luci/appengine/swarming/server/task_scheduler.py;l\u003d776-778;drc\u003dbb840bc41d72df96b0f84898edd5ab7ce46e9326",
      "parentUuid": "2e01ddb9_a52db474",
      "revId": "b2b70ab449353f15816867f1932800d11d66a66e",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba"
    }
  ]
}